src/decorators/produces.ts

```diff
@@ -0,0 +1,3 @@
+export function Produces(mediaType: string): any {
```

Comment:
Shouldn’t string be a string literal of all of the possible media types that are allowed?

Reply:
There are approx. [2500 different media types](https://www.iana.org/assignments/media-types/media-types.xhtml). Doesn't seem to be practical to include them all as a union type and then maintain the list. And providing only a subset is definitely worse than just allowing any string, leaving the control to the user.

Reply:
Good point. I’ll mark this comment as resolved.